feature/experimental/route_redistribution/feature.textproto

```diff
@@ -68,16 +68,16 @@ telemetry_path {
 
 
 feature_profile_dependency {
-  name: "bgp_base"
+  name: "bgp"
   version: 1
 }
 
 feature_profile_dependency {
-  name: "local_aggregates"
+  name: "experimental_route_redistribution"
   version: 1
 }
 
 feature_profile_dependency {
-  name: "bgp_policy_base"
+  name: "experimental_policy_policy_base"
```

Comment:
should be bgp_policybase?

Reply:
Shouldn't `bgp_policy_base` be changed to `bgp_policybase`? Or was it an error for this to depend on the BGP feature?

Reply:
Sorry, it looks like this did get fixed.